### PR TITLE
QPPA-798: Add DM_COMPOSITE measure

### PR DESCRIPTION
**Pull request by marimiyachi**
JIRA issue: https://jira.cms.gov/browse/QPPA-798

Summary: 
* Adds DM_COMPOSITE measure as described in the ticket
* Removes cmsWebInterface as possible submission methods for measure '001' and '117'
* Removes the `/data` directory and reorganizes the `/util` and `/scripts` directories. `/util` contains all supplementary data and util methods. `/scripts` contains all methods for generating data.
* Other small refactors to clean up a bit and remove bloated code

Tests are passing.

Reviewer: @kencheeto

### CODE REVIEW

File: test/measures/measures-data-spec.js [hunk 1]
```diff
@@ -54,14 +54,23 @@ describe('measures data json', function() {
...
+      const intersection = new Set([...qualityMeasureIds].filter(x => multiPerformanceIds.has(x)));
 
       assert.equal(intersection.size, multiPerformanceIds.size);
     });
 
+    it('properly handles the exclusion of certain submission methods', () => {
+      const shouldNotAcceptCmsWebInterface = measuresData.filter(
+        measure => measure.measureId === '001' || measure.measureId === '117'
+      );
+      _.each(shouldNotAcceptCmsWebInterface, (measure) => {
```
Comment (kencheeto) [inline test/measures/measures-data-spec.js]:
```js
shouldNotAcceptCmsWebInterface.forEach(measure => {
```

File: test/measures/measures-data-spec.js [hunk 2]
```diff
@@ -54,14 +54,23 @@ describe('measures data json', function() {
...
+        assert.isFalse(_.includes(measure.submissionMethods, 'cmsWebInterface'));
```
Comment (kencheeto) [inline test/measures/measures-data-spec.js]:
```js
assert.isFalse(measure.submissionMethods.includes('cmsWebInterface'));
```

File: test/measures/measures-data-spec.js [hunk 3]
```diff
@@ -54,14 +54,23 @@ describe('measures data json', function() {
...
+      const multiPerformanceIds = new Set(['007', '046', '122', '238', '348', '391', '392', '394', '398']);
+      const qualityMeasureIds = _.map(_.filter(measuresData, {category: 'quality'}), 'measureId');
+      const intersection = new Set([...qualityMeasureIds].filter(x => multiPerformanceIds.has(x)));
 
       assert.equal(intersection.size, multiPerformanceIds.size);
     });
 
+    it('properly handles the exclusion of certain submission methods', () => {
+      const shouldNotAcceptCmsWebInterface = measuresData.filter(
+        measure => measure.measureId === '001' || measure.measureId === '117'
```
Comment (kencheeto) [inline test/measures/measures-data-spec.js]:
```js
measure => ['001', '117'].includes(measure.measureId)
```

File: test/measures/measures-data-spec.js [hunk 4]
```diff
@@ -54,14 +54,23 @@ describe('measures data json', function() {
   });
 
   describe('quality measures', function() {
-    it('includes all quality measures with mutli-performance strata', function() {
-      var multiPerformanceIds = new Set(['007', '046', '122', '238', '348', '391', '392', '394', '398']);
-      var qualityMeasureIds = _.map(_.filter(measuresData, {category: 'quality'}), 'measureId');
-      var intersection = new Set([...qualityMeasureIds].filter(x => multiPerformanceIds.has(x)));
+    it('includes all quality measures with multi-performance strata', function() {
```
Comment (kencheeto) [inline test/measures/measures-data-spec.js]: 🙏

File: test/measures/measures-data-spec.js (hunk 3 above)
Comment (marimiyachi) [inline test/measures/measures-data-spec.js]: nice

File: test/measures/measures-data-spec.js (hunk 1 above)
Comment (marimiyachi) [inline test/measures/measures-data-spec.js]: done!

File: test/measures/measures-data-spec.js (hunk 2 above)
Comment (marimiyachi) [inline test/measures/measures-data-spec.js]: done!

Comment (kencheeto) [inline test/measures/measures-data-spec.js]: aw, i just found `assert.notInclude` - TIL

```js
assert.notInclude(measure.submissionMethods, 'cmsWebInterface');
```

**Pull request closed without merging**